ctapipe/containers.py

```diff
@@ -843,6 +849,19 @@ class ParticleClassificationContainer(Container):
     telescopes = Field(None, "Telescopes used if stereo, or None if Mono")
 
 
+class DispContainer(Container):
+    """
+    Standard output of disp reconstruction algorithms for origin reconstruction
+    """
+
+    default_prefix = "disp"
+
+    norm = Field(nan * u.deg, "reconstructed absolute value for disp", unit=u.deg)
+    sign = Field(nan, "reconstructed sign for disp")
+    # "parameters_is_valid" instead of "is_valid" to avoid duplication of "disp_is_valid" column
+    parameters_is_valid = Field(False, "true if the predictions are valid")
```

Comment:
Can you explain what the problem is?  You should have both a reconstructor prefix and the container prefix, so I don't see how you would get a name collision.  I don't see any need to change the reconstruction container names.

Reply:
At the moment we fill the reconstructor name / model name as the prefix.

I agree that this issue should be solved with a different prefix, not with a custom suffix on the field name

Reply:
Also, this should be `parameters_are_valid` to be grammatically correct (or `disp_is_valid`) 

is the issue only when reading from a Table I guess? Is it a problem with mixed telescope vs subarray columns? There I can see you might have name collisions if he prefixes are not set right.   We had this same issue with per-tel columns mixed with subarray columns before, which is why we added `_tel_`.

Reply:
The problem here is that `DispParameters` and `ReconstructedGeometry` get assigned the same prefix, which makes them collide when loaded by `TableLoader` into a single table.

Reply:
This could actually be solved by having `TableLoader` include the full path, e.g. `dl2.event.tel.geometry.DispReconstructor.is_valid` but that will make for some pretty long column names...

Reply:
Maybe just include the container name in the prefix?